src/utils/currencies.ts

```diff
@@ -4,14 +4,14 @@ export enum Currency {
   Euro = 'cEUR',
 }
 
-// Important: when adding new currencies, the string must match the symbol
-// we use in address-metadata
+/**
+ * @deprecated Should use tokenIds / symbols from token list instead.
+ */
 export enum CiCoCurrency {
   CELO = 'CELO',
   cUSD = 'cUSD',
   cEUR = 'cEUR',
   cREAL = 'cREAL',
-  ETH = 'ETH',
```

Comment:
to make knip happy. This wasn't used anywhere. We should ultimately get rid of this, but there are some legacy fiat exchanges dependencies on this